src/code/components/animated-sprite.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+const tweenState = require('react-tween-state');
```

Comment:
Thank you for this Kirk! I was wondering how to add the package. Also moved the CSS changes into an `fv-layout-c` section so now all layouts still look good.